lib/check-self-update/parse-json.js

```diff
@@ -0,0 +1,11 @@
+/* Sometimes, a json can be malformed. This ensures error-handling */
+
+const parseJson = (string) => {
```

Comment:
@rafaelcalpena The parse-json module duplicates https://github.com/LabShare/lsc/blob/master/lib/cli/utils.js#L38.